edge/pkg/devicetwin/dtmodule/dtmodule_test.go

```diff
@@ -17,12 +17,17 @@ limitations under the License.
 package dtmodule_test
 
 import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
+	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtcommon"
 	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtcontext"
+	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtmanager"
 	. "github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtmanager"
 	. "github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtmodule"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"testing"
+	"time"
```

Comment:
please format imports.
I don't think it's necessary to use `. "github.com/kubeedge/kubeedge/edge/pkg/devicetwin/<dtmanager, dtmanager>"`

Reply:
Will do.